### ISSUE TITLE
[HOLD #42214] Report/expenses don’t show in the workspace chat

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#42090]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.73-0
**Reproducible in staging?:** Yes
**Reproducible in production?:** Yes
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @garrettmknight 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1715595596241419

## Action Performed:
1. Launch the app and login as employee
2. Scan and submit an expense in workspace chat
## Expected Result:
Expenses and report are visible in workspace and able to view the expenses in report chat

## Actual Result:
The expenses and report aren’t visible in the workspace chat and when viewed the report thread via OldDot report history, the report chat only shows the total and not the expenses/receipts.

## Workaround:
Unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [X] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence
![Approver report view](https://github.com/Expensify/App/assets/38435837/1d68e77e-3e5d-4208-9334-2406ff3d7ffa)
![Submitter report view](https://github.com/Expensify/App/assets/38435837/e7b6eeba-5d24-47fc-b60c-8363b0473ad9)
![Old dot thread view](https://github.com/Expensify/App/assets/38435837/4232b4cc-6438-47d9-b540-141f86c1b55c)

https://github.com/Expensify/App/assets/38435837/a8573ecb-1f10-4652-a112-461b43588194

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by Beamanator** [comment]
Asking if this is currently reproducible in prod, since this was reported in v1.4.72-1 in slack

**Comment by Beamanator** [comment]
@garrettmknight confirmed the bug exists in prod but not in staging (see [slack](https://expensify.slack.com/archives/C049HHMV9SM/p1715688090316169?thread_ts=1715595596.241419&cid=C049HHMV9SM)) so this shouldn't be a deploy blocker)

**Comment by JmillsExpensify** [comment]
Still experiencing this bug

**Comment by puneetlath** [comment]
I haven't seen this lately. @JmillsExpensify you're still actively experiencing it?

**Comment by puneetlath** [comment]
Asked here: https://expensify.slack.com/archives/C049HHMV9SM/p1717027756037069?thread_ts=1715595596.241419&cid=C049HHMV9SM

**Comment by puneetlath** [comment]
Unclear whether it is still happening. Waiting on more reports.

**Comment by puneetlath** [comment]
Seems to most likely be the same issue as https://github.com/Expensify/App/issues/42214 so I'm going to put it on hold for that.

**Comment by puneetlath** [comment]
Still on hold.
